core/src/main/codegen/templates/Parser.jj

```diff
@@ -4033,6 +4037,29 @@ SqlNode SqlExec() :
     }
 }
 </#if>
+<#if parser.usingStatementParserMethods?size != 0>
+/**
+ * Parses a Using statement.
+ */
+SqlNode SqlUsing() :
+{
+    final Span s;
+    SqlNode source;
+}
+{
+    <USING> { s = span(); }
+    (
+<#-- additional literal parser methods are included here -->
+<#list parser.usingStatementParserMethods as method>
+        source =  ${method}(s)
+        <#sep>|</#sep>
```

Comment:
Also a question about JavaCC and this project here. Are you expecting a use case of multiple USING statements? Since there is only one method in parser.usingStatementParserMethods , is this separation necessary? Are you expecting the number of methods to grow?

Reply:
I don't have enough domain knowledge to know if the list will grow but it is possible I suppose. I'm in the same boat as you in this case and have just been following what other methods do as a base for my designs.

Reply:
Since the methods are added using a list syntax in config.fmpp, it seems like best practice to write the corresponding logic in Parser.jj to handle the case of elements being added to that list in the future.

Otherwise, a developer could resonably add a new entry into the list in config.fmpp and be confused why the code no longer compiles.